ui/style.css

```diff
@@ -324,3 +358,24 @@ blockquote {
 	75% { background-color: black; }
 	100% { background-color: red; }
 }
+
+/* make @ symbol always in handle input field. */
```

Comment:
THis is a nice change. Makes things in that input box a lot clearer (e.g. not to actually include the @ in your text)

Reply:
yah, @qubist did that.